src/main.cpp

```diff
@@ -365,8 +369,7 @@ int main(int argc, char* argv[])
      *******************************************************************/
 
 #if defined(Q_OS_WIN) && defined(INCLUDE_UPDATER)
-    auto abortLaunch = runUpdate();
-    if (abortLaunch) {
+    if (runUpdate()) {
```

Comment:
I'd like to keep this code - just reading "if runupdate is true" makes no sense, but if "if abort launch is true" does

Reply:
Okay, it just seemed sensible to eliminate an unneeded local `bool abortLaunch` variable that is not used beyond this point in `main.cpp` ... - I'll put it back then in the interests of code clarity.